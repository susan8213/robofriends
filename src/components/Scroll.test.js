import { shallow } from 'enzyme';
import React from 'react';
import Scroll from './Scroll';


describe('<Scroll />', () =>{

    it('render without crashing', () => {
        expect(shallow(<Scroll/>).html()).toMatchSnapshot();
    })


})