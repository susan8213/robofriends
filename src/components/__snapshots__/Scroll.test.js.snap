// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scroll /> render without crashing 1`] = `"<div style=\\"overflow:scroll;height:80vh\\"></div>"`;
